src/mmrelay/radio/backends/meshtastic_backend.py

```
@@ -0,0 +1,320 @@
+                close_timed_out = False
+                try:
+                    future.result(timeout=10.0)  # 10-second timeout
+                except concurrent.futures.TimeoutError:
+                    close_timed_out = True
+                    meshtastic_logger.warning(
+                        "Meshtastic client close timed out - may cause notification errors"
+                    )
+                    # Best-effort cancellation; the underlying close may be
+                    # stuck in BLE/DBus, but we cannot block shutdown.
+                    future.cancel()
+                except Exception:  # noqa: BLE001 - shutdown must keep going
```

Ruff (0.14.14)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)